src/views/Students/StudentComponents/AlumniService.jsx

```diff
@@ -107,6 +109,20 @@ const AlumniService = (props) => {
                 <DetailField label="Location" value={alumniService.location} />
                 <DetailField label="End Date" value={alumniService.end_date ? moment(alumniService.end_date).format("DD MMM YYYY") : ''} />
                 <DetailField label="Fee Amount" value={alumniService.fee_amount} />
+                <DetailField label="Program Mode" value={alumniService.program_mode}/>
+                <DetailField label="Category" value={alumniService.category}/>
+                {selectedCategory === "Youth Leadership" && (
+                  <DetailField label={alumniService.category} value={alumniService.youth_leadership}/>
+                )}
+                {selectedCategory === "Peer-Learning" && (
```

Comment:
shouldn't this be
```suggestion
                {selectedCategory === "Peer Learning" && (
```

why is there an hyphen in between?

Reply:
done changes.